interface/forms/eye_mag/js/eye_base.php

```diff
@@ -3946,12 +3944,11 @@ function show_by_setting() {
                   $('#left-panel').css("right","0px");
                   $('#EXAM_KB').css({position: 'fixed', top: '29px'});
                   $('#EXAM_KB').css('display', 'block');
-                  $('#EXAM_KB').draggable();
-                  $('#IMP').droppable({ drop: dragto_IMPPLAN } );
-                  $('#IMPPLAN_zone').droppable({ drop: dragto_IMPPLAN_zone } );
-                  $('#IMPPLAN_text').droppable({ drop: dragto_IMPPLAN_zone } );
-
-                  $('[id^="PLANS"]').draggable(  { cursor: 'move', revert: true });
+                  draggable('#EXAM_KB');
+                  //$('#IMP').droppable({ drop: dragto_IMPPLAN } );
+                  //$('#IMPPLAN_zone').droppable({ drop: dragto_IMPPLAN_zone } );
+                  //$('#IMPPLAN_text').droppable({ drop: dragto_IMPPLAN_zone } );
```

Comment:
recommend removing rather than comment out code

Reply:
Updated the commit.